docs/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This is good -- go ahead and remove the .DS_Store files, too; I think it's 'git rm .DS_Store' ?

Reply:
Will do